docs/Configuration.md

```diff
@@ -40,35 +40,60 @@ Usage of openvpn-auth-oauth2:
       --version                              shows versions
 ```
 
-## Setup HTTP listener
-
+## Configuration openvpn-auth-oauth2
 openvpn-auth-oauth2 starts a http listener which needs to be accessible from OpenVPN client before the VPN connection is established.
 By default, the http listener runs on :9000.
 
 It's highly recommend to put openvpn-auth-oauth2 behind a reverse proxy which terminates the TLS connections. It's important to configure
 `CONFIG_HTTP_BASE_URL` because openvpn-auth-oauth2 need to know the redirect url.
 
-### Configuration
-
-- `CONFIG_HTTP_LISTEN=:9000`
-- `CONFIG_HTTP_BASE_URL=https://login.example.com`
-
-## Setup OIDC Provider
-
-See [Providers](Providers) for more information
+Example:
+```conf
+# openvpn-auth-oauth2 config file
+CONFIG_HTTP_LISTEN=:9000
+CONFIG_HTTP_BASE_URL=https://login.example.com
+```
 
 ## Setup OpenVPN server
+To connect openvpn-auth-oauth2 with openvpn server add lines below:
 
-### server.conf
-
-```
+```conf
+# openvpn server.conf
+...
 # /etc/openvpn/password.txt is a password file where the password must be on first line
 management /run/openvpn/server.sock unix /etc/openvpn/password.txt
 management-hold
 management-client-auth
 ```
 
-### Configuration
+```conf
+# openvpn-auth-oauth2 config file
+CONFIG_OPENVPN_ADDR=unix:///run/openvpn/server.sock
+CONFIG_OPENVPN_PASSWORD=<password>
+```
+
+## Setup OIDC Provider
+
+See [Providers](Providers) for more information
+
+## Full configuration example
+Configuration openvpn-auth-oauth2 for zitadel
 
-- `CONFIG_OPENVPN_ADDR=unix:///run/openvpn/server.sock`
-- `CONFIG_OPENVPN_PASSWORD=<password>`
+```conf
+# Define the public http endpoint here.
+CONFIG_HTTP_BASEURL=http://<vpn>:9000/
+CONFIG_HTTP_LISTEN=:9000
+# Define a random value with 16 or 24 characters
+CONFIG_HTTP_SECRET=1jd93h5b6s82lf03jh5b2hf9
+CONFIG_OPENVPN_ADDR=unix:///run/openvpn/server.sock
+CONFIG_OPENVPN_PASSWORD=<password from /etc/openvpn/password.txt>
+CONFIG_OAUTH2_ISSUER=https://company.zitadel.cloud
+CONFIG_OAUTH2_ENDPOINT_AUTH=https://company.zitadel.cloud/oauth/v2/authorize
+CONFIG_OAUTH2_ENDPOINT_TOKEN=https://company.zitadel.cloud/oauth/v2/token
+CONFIG_OAUTH2_ENDPOINT_DISCOVERY=https://company.zitadel.cloud/.well-known/openid-configuration
```

Comment:
Does auto-discovery not work? Normally, define the CONFIG_OAUTH2_ISSUER should be good enough.